pkg/controller/nodenetworkstate/conditions.go

```diff
@@ -0,0 +1,48 @@
+package nodenetworkstate
+
+import (
+	"time"
+
+	nmstatev1 "github.com/nmstate/kubernetes-nmstate/pkg/apis/nmstate/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func SetCondition(instance *nmstatev1.NodeNetworkState, conditionType nmstatev1.NodeNetworkStateConditionType, status corev1.ConditionStatus, reason, message string) {
```

Comment:
To use instance as receiver I would have to move it to API because I cannot define methods on non-local type. In API we don't have any functionality. What do you think should I move it there or not?